### ISSUE TITLE
New rule: no-test-only

### ISSUE THREAD

**Issue by aladdin-add** [not-an-aardvark/eslint-plugin-eslint-plugin#118]
refs: https://github.com/eslint/rfcs/tree/main/designs/2020-rule-tester-only
RuleTester test cases support a new only property to run an individual case or subset of cases for less noisy debugging. It should be only used in development.

```js
valid: [
{ code: "", only: true}, // should report an error
]
```

```js
valid: [
ruleTester.only("var i = 0;"), // should report an error
]
```

**Comment by aladdin-add** [comment]
it has been released in eslint v7.29.0. refs: https://eslint.org/blog/2021/06/eslint-v7.29.0-released
